site/diagrams/sequenceDiagrams/connectToCredentials.txt

```diff
@@ -0,0 +1,6 @@
+title Connection to Credentials API
+
+Client->Credentials: 1 open
+
+Client->Credentials: 2 attach("sender", "credentials/${tenant_id}")
+Client->Credentials: 3 attach("receiver", "credentials/${tenant_id}/${reply-to}")
```

Comment:
missing:
`Credentials->Client: 6 attach("sender", "credentials/${tenant_id}/${reply-to}")`